config/parser.go

```diff
@@ -0,0 +1,103 @@
+package config
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+)
+
+// Config describes config options
+type Config struct {
+	Db       *Db
+	Email    *Email
+	Sreality *Sreality
+}
+
+// Db describes config options for DB
+type Db struct {
+	Engine   string `json:"engine"`
+	Host     string `json:"host"`
+	Port     int    `json:"port"`
+	Username string `json:"username"`
+	Password string `json:"password"`
+	Database string `json:"database"`
+}
+
+// Email describes config options for email sending
+type Email struct {
+	To       string `json:"to"`
+	From     string `json:"from"`
+	Server   string `json:"server"`
+	TLS      bool   `json:"tls"`
+	TLSPort  int    `json:"tls_port"`
+	Username string `json:"username"`
+	Password string `json:"password"`
+}
+
+// Sreality describes config options for the flat search request
+type Sreality struct {
```

Comment:
If you want to have sreality as a separate package (and used by someone elsse), keep this config in that sreality package.